src/main/java/seedu/address/newstorage/StorageManager.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.newstorage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.model.ReadOnlyRecipeBook;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.UserPrefs;
+/**
+ * Manages storage of AddressBook data in local storage.
```

Comment:
minor: missed one JavaDocs to update

```suggestion
 * Manages storage of RecipeBook data in local storage.
```